resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosCoarseGrainedSchedulerBackend.scala

```diff
@@ -308,9 +312,17 @@ private[spark] class MesosCoarseGrainedSchedulerBackend(
     val mem = getResource(offer.getResourcesList, "mem")
     val cpus = getResource(offer.getResourcesList, "cpus")
     val ports = getRangeResource(offer.getResourcesList, "ports")
+    val unavailabilityStartMs = offer.hasUnavailability match {
```

Comment:
no longer ms since you have `toString`, so please use just `unavailabilityStart`